src/elfpy/agent.py

```diff
@@ -18,7 +18,12 @@ class Agent:
     date value is an inte with how many tokens they have for that date
     """
 
-    def __init__(self, market: Market, rng: Generator, wallet_address: int, budget: float):
+    # pylint: disable=too-many-instance-attributes
```

Comment:
If you're going to add pylint disables, please also add descriptions of the problem and a path to fix them. We should avoid disabling these if possible.

Reply:
removed, not needed any more